sql/telemetry_derived/deviations_v1/query.sql

```diff
@@ -5,7 +5,7 @@ SELECT
   geography,
   metric
 FROM
-  `moz-fx-data-shared-prod.analysis.deviations`
+  `moz-fx-data-shared-prod.telemetry_derived.deviations_v1`
```

Comment:
```suggestion
  deviations_v1
```

Per style guidelines, we should not fully qualify the table name here.